app/scripts/pages/files.js

```diff
@@ -48,14 +46,12 @@ export default React.createClass({
 
   addFile: function (e) {
     e.preventDefault()
-    $(ReactDOM.findDOMNode(this)).find('.file-select').click()
-    return
+    this.refs.fileSelect.click()
```

Comment:
Honestly not sure if it's required... I didn't check lol. I just changed it to not use jquery. What it's doing though is the button exposed to click is just a regular button, and the file input is hidden. Without this I'm taking a guess that you will only be able to drag and drop.

There might be a way around, but nothing that I know of off the top of my head.